### DISCUSSION TITLE
v3.13.2 不能复制阿里云盘到115网盘。日志：

### DISCUSSION THREAD

**Discussion by yoyo3287258** [AlistGo/alist#3790] [General]
ERRO[2023-03-10 09:27:09] error [unexpected error
github.com/SheltonZhu/115driver/pkg/driver.init
	/go/pkg/mod/github.com/!shelton!zhu/115driver@v1.0.13/pkg/driver/error.go:13
runtime.doInit
	/usr/local/go/src/runtime/proc.go:6506
runtime.doInit
	/usr/local/go/src/runtime/proc.go:6483
runtime.doInit
	/usr/local/go/src/runtime/proc.go:6483
runtime.doInit
	/usr/local/go/src/runtime/proc.go:6483
runtime.doInit
	/usr/local/go/src/runtime/proc.go:6483
runtime.main
	/usr/local/go/src/runtime/proc.go:233
runtime.goexit
	/usr/local/go/src/runtime/asm_arm64.s:1172
请升级到最新版本
github.com/SheltonZhu/115driver/pkg/driver.GetErr
	/go/pkg/mod/github.com/!shelton!zhu/115driver@v1.0.13/pkg/driver/error.go:131
github.com/SheltonZhu/115driver/pkg/driver.(*UploadInitResp).Err
	/go/pkg/mod/github.com/!shelton!zhu/115driver@v1.0.13/pkg/driver/response.go:173
github.com/SheltonZhu/115driver/pkg/driver.CheckErr
	/go/pkg/mod/github.com/!shelton!zhu/115driver@v1.0.13/pkg/driver/error.go:142
github.com/SheltonZhu/115driver/pkg/driver.(*Pan115Client).UploadSHA1
	/go/pkg/mod/github.com/!shelton!zhu/115driver@v1.0.13/pkg/driver/upload.go:223
github.com/SheltonZhu/115driver/pkg/driver.(*Pan115Client).UploadFastOrByMultipart
	/go/pkg/mod/github.com/!shelton!zhu/115driver@v1.0.13/pkg/driver/upload.go:267
github.com/alist-org/alist/v3/drivers/115.(*Pan115).Put
	/source/drivers/115/driver.go:90
github.com/alist-org/alist/v3/internal/op.Put
	/source/internal/op/fs.go:546
github.com/alist-org/alist/v3/internal/fs.copyFileBetween2Storages
	/source/internal/fs/copy.go:98
github.com/alist-org/alist/v3/internal/fs.copyBetween2Storages.func2
	/source/internal/fs/copy.go:76
github.com/alist-org/alist/v3/pkg/task.(*Task[...]).run
	/source/pkg/task/task.go:75
github.com/alist-org/alist/v3/pkg/task.(*Manager[...]).do.func1
	/source/pkg/task/manager.go:33
runtime.goexit
	/usr/local/go/src/runtime/asm_arm64.s:1172
github.com/alist-org/alist/v3/internal/op.Put
	/source/internal/op/fs.go:569
github.com/alist-org/alist/v3/internal/fs.copyFileBetween2Storages
	/source/internal/fs/copy.go:98
github.com/alist-org/alist/v3/internal/fs.copyBetween2Storages.func2
	/source/internal/fs/copy.go:76
github.com/alist-org/alist/v3/pkg/task.(*Task[...]).run
	/source/pkg/task/task.go:75
github.com/alist-org/alist/v3/pkg/task.(*Manager[...]).do.func1
	/source/pkg/task/manager.go:33
runtime.goexit
	/usr/local/go/src/runtime/asm_arm64.s:1172] while run task [copy [/ali](/资料/【深蓝学院-99】机器学习数学基础/任务4-2：【视频】第二章：BP算法（上）.mp4) to [/115](/资料/计算机技术视频/【深蓝学院-99】机器学习数学基础)]

**Comment by charleycy** [comment]
同问，也遇到了
